flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBStateBackend.java

```diff
@@ -686,6 +729,13 @@ public RocksDBNativeMetricOptions getMemoryWatcherOptions() {
 		return options;
 	}
 
+	/**
+	 * Gets the thread number will used for downloading files from DFS when restore.
+	 */
+	public int getRestoredThreadNum() {
+		return restoredThreadNum;
```

Comment:
here we could return `restoredThreadNum` if it is not -1 (defined) and `CheckpointingOptions.CHECKPOINT_RESTORE_THREAD_NUM.defaultValue()` if `restoredThreadNum` is still -1 (undefined).